src/rosdiscover/interpreter/context.py

```diff
@@ -234,6 +234,13 @@ def action_client(self, ns: str, fmt: str) -> None:
             self.sub(f'{ns}/feedback', f'{fmt}Feedback', implicit=True)
             self.sub(f'{ns}/result', f'{fmt}Result', implicit=True)
 
+    def actions_have_topics(self):
+        distribution = self.app.description.distribution
+        if distribution == ROSVersion.ROS1:
+            return True
+        else:
+            return distribution.ros == ROSVersion.ROS2 and distribution.name < ROSDistribution.FOXY
```

Comment:
These changes are good, but I don't think that `distribution.name < ROSDistribution.FOXY` should work. Really, we should add a `__lt__` or `__cmp__` method to `ROSDistribution`, and then we can just do `distribution < ROSDistribution.Foxy`. I'll add that to ROSWire now.

Reply:
Done.